src/Cli/WorkerCommand.php

```diff
@@ -0,0 +1,164 @@
+
+    /**
+     * @var HttpClient
+     */
+    private $httpClient;
+
+    /**
+     * @param SqsClient  $sqsClient
+     * @param HttpClient $httpClient
+     */
+    public function __construct(SqsClient $sqsClient, HttpClient $httpClient)
+    {
+        $this->sqsClient  = $sqsClient;
+        $this->httpClient = $httpClient;
+
+        parent::__construct();
+    }
+
+    /**
+     * Configure the command
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('eb-worker')
+            ->setDescription('Launch the local Elastic Beanstalk worker')
+            ->addOption(
+                'server',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Server to which requests are redirected'
+            )
+            ->addOption(
+                'queue',
+                null,
+                InputOption::VALUE_REQUIRED,
+                'Queue name to pull messages from'
+            );
+    }
+
+    /**
+     * Executes the current command
+     *
+     * @param  InputInterface  $input
+     * @param  OutputInterface $output
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $server    = $input->getOption('server');
+        $queueName = $input->getOption('queue');
+
+        try {
+            $queueUrl = $this->sqsClient->getQueueUrl(['QueueName' => $queueName])['QueueUrl'];
+        } catch (SqsException $exception) {
+            $output->writeln(sprintf(
+                '<error>Impossible to retrieve URL for queue "%s". Reason: %s</error>',
+                $queueName,
+                $exception->getMessage()
+            ));
+
+            return;
+        }
+
+        while (true) {
+            $messages = $this->sqsClient->receiveMessage([
+                'QueueUrl'            => $queueUrl,
+                'AttributeNames'      => ['All'],
+                'MaxNumberOfMessages' => 1,
+                'WaitTimeSeconds'     => 20
+            ]);
+
+            if (!$messages->hasKey('Messages')) {
+                continue;
+            }
+
+            $this->processMessage($messages['Messages'][0], $server, $queueName, $queueUrl, $output);
+        }
+    }
+
+    /**
+     * Process a single message and push it to the server URL
+     *
+     * @link   http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features-managing-env-tiers.html
+     * @param  array           $message
+     * @param  string          $server
+     * @param  string          $queueName
+     * @param  string          $queueUrl
+     * @param  OutputInterface $output
+     * @return void
+     */
+    private function processMessage(
+        array $message,
+        string $server,
+        string $queueName,
+        string $queueUrl,
+        OutputInterface $output
+    ) {
+        $response = $this->httpClient->post(rtrim($server, '/') . '/internal/worker', [
```

Comment:
Why don't you let the user specify the full URL via argument? WorkerMiddleware works in any URL.